model-api/src/commonMain/kotlin/org/modelix/model/data/MetaModelData.kt

```diff
@@ -34,6 +35,20 @@ data class ConceptData(
     val extends: List<String> = emptyList()
 )
 
+@Serializable
+data class EnumData(
```

Comment:
MPS allows to mark one of the enum members as default (it is return if the property is not set). This information is  missing.